ignition/service/messaging.py

```diff
@@ -192,6 +193,10 @@ def post(self, envelope, key=None):
             self.delivery_service.deliver(envelope, key=key)
 
 class KafkaDeliveryService(Service, DeliveryCapability):
+    
+    def sigterm_handler(self, sig, frame):
+        logger.debug('sigterm_handler')
```

Comment:
Can this log message express more detail? E.g. `KafkaDeliveryService sigterm_handler triggered. Closing Kafka producer`.